synfig-studio/configure.ac

```diff
@@ -221,6 +221,7 @@ AC_CONFIG_FILES([
     src/gui/Makefile
     src/gui/resources/Makefile
     src/gui/resources/css/Makefile
+	src/gui/resources/ui/Makefile
```

Comment:
Annoying but ice0 agreed to use tabs only for Synfig C++ and CMake code

Autotools :(

So please change that line indented with tab to spaces.

Reply:
@Keyikedalube Done :)

Reply:
Resolved.